### ISSUE TITLE
can you add mxc's  api?

### ISSUE THREAD

**Issue by 317459062** [ccxt/ccxt#4823]
ATTENTION!!!

MUST READ THIS BEFORE SUBMITTING ISSUES:

https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-submit-an-issue

- OS:windows
- Programming Language version:python3.5
- CCXT version:1.18.76
- Exchange:mxc
- Method:all

**Comment by 317459062** [comment]
mxc website:       https://www.mxc.com/
mxc's api detail   https://www.mxc.com/open/swagger-ui.html#/----

**Comment by 317459062** [comment]
ok thanks  

how long will take it

**Comment by kroitor** [comment]
@317459062 we can't give you an estimate, unfortunately, due to reasons explained here: https://github.com/ccxt/ccxt/wiki/FAQ, but we'll do our best to add it as soon as possible.

**Comment by xnys** [comment]
@kroitor Looking forward to the early completion of this exchange, I have been following it for a long time and have not made any progress. Thank you.

**Comment by katopway** [comment]
@kroitor Is there any specific plan on this work? Looking forward for any progress on this.  Thanks a lot.

**Comment by kroitor** [comment]
@katopway yes, we will get back to active work on integrations after releasing CCXT Pro, which is within reach now.

**Comment by lfzCarlosC** [comment]
please announce when it is ready

**Comment by katopway** [comment]
@kroitor Is there any work done on this?

**Comment by kroitor** [comment]
@katopway you may want to check out this PR: https://github.com/ccxt/ccxt/pull/6308

**Comment by katopway** [comment]
@kroitor thanks a lot
